swervelib/ctre/KrakenX60DriveControllerFactoryBuilder.java

```diff
@@ -15,12 +15,15 @@ public final class KrakenX60DriveControllerFactoryBuilder {
 
     private static final int CAN_TIMEOUT_MS = 250; 
     private static final int STATUS_FRAME_GENERAL_PERIOD_MS = 250;
+    private static final double ENCODER_CPR = 1;
```

Comment:
2048, not 1 - https://pro.docs.ctr-electronics.com/en/stable/docs/migration/migration-guide/closed-loop-guide.html#

Reply:
Matter of fact, why are we adding ENCODER_CPR - isn't TICKS_PER_ROTATION already there?